### ISSUE TITLE
Duplicate characters and migration not working

### ISSUE THREAD

**Issue by znbaboy** [JannikBirn/palworld-host-save-fix#30]
[Level.zip](https://github.com/JannikBirn/palworld-host-save-fix/files/14267605/Level.zip)

i was able to migrate my friends with no issues, but for me I ended up with two characters with the same name. Zurk level 45 (my main) and 0 (one i want deleted).

if I migrate my level 45 save to my level 0 save, i have to made a new character.

Not sure whats going on here, but this is causing issues when I try removing the character using this tool: https://github.com/magicbear/palworld-server-toolkit

**Comment by JannikBirn** [comment]
> if I migrate my level 45 save to my level 0 save, i have to made a new character.

That's sound really strange, are you sure the migration succeeded? There should be a pop-up in the end if you are using the gui.py or the executable. If it just closes without that pop-up an error might occurred.

**Comment by znbaboy** [comment]
> > if I migrate my level 45 save to my level 0 save, i have to made a new character.
> 
> That's sound really strange, are you sure the migration succeeded? There should be a pop-up in the end if you are using the gui.py or the executable. If it just closes without that pop-up an error might occurred.

yes it was successful. we have been playing since no issues. only now when i want to remove my old characters am i noticing a problem

**Comment by JannikBirn** [comment]
Alright, I'll take a look ASAP and check why the old player wasn't removed correctly.

**Comment by znbaboy** [comment]
i go into a bit more detail in this issue post: https://github.com/magicbear/palworld-server-toolkit/issues/38
